data/migrations/20190812100226_create-cars.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('users', tbl => {
+        tbl.increments();
+        tbl.text('VIN',99)
+            .unique()
+            .notNullable();
+        tbl.text('make', 99)
+            .notNullable();
+        tbl.text('model')
+            .notNullable();
+        tbl.integer('mileage')
+            .notNullable();
+    });
```

Comment:
don't forget to make a column for transmission type and title status. It's not required for the data to pass to the database, but it is required for our table so that we can pass that data.